modules/extension/wmts/src/main/java/org/geotools/ows/wmts/map/WMTSMapLayer.java

```diff
@@ -123,12 +125,14 @@ public boolean isNativelySupported(CoordinateReferenceSystem crs) {
         return getReader().isNativelySupported(crs);
     }
 
+    @Deprecated
     public String getRawTime() {
         return rawTime;
     }
 
+    /** @deprecated - Should be passed in by parameters */
+    @Deprecated
```

Comment:
This deprecation is going to break GeoServer build, it uses setRawTime in GetMap.java

Reply:
Btw, I cannot find how the time value is managed now, is there an example? I can only see a bit that sets a default value for the time, but no place where an actual time value is provided from outside.

Reply:
Sorry about that one, I was a little bit too eager to get rid of it. I mean we should send this in by parameter, but then we should make it more generic. Having support for whatever dimension the server can offer. I will reintroduce everything regarding the requestedTime.